internal/rbd/rbd_healer.go

```diff
@@ -0,0 +1,184 @@
+/*
+Copyright 2021 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rbd
+
+import (
+	"context"
+	"errors"
+
+	"github.com/container-storage-interface/spec/lib/go/csi"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8s "k8s.io/client-go/kubernetes"
+
+	"github.com/ceph/ceph-csi/internal/util"
+)
+
+func accessModeStrToInt(mode string) csi.VolumeCapability_AccessMode_Mode {
+	switch {
```

Comment:
just use

```go
switch mode {
case "ReadWriteOnce":
    ...
```

But I'm surprised that there is no existing helper function for this yet..

Reply:
Indeed, I searched the repo assuming there should be at least something similar.

I took the above snippet from some Kube code and pasted it here.
Sure the one you suggested looks simplified, I will fix it.